vamb/__main__.py

```diff
@@ -2511,6 +2640,14 @@ def main():
     add_composition_npz_argument(abundance_parser)
     add_abundance_args_nonpz(abundance_parser)
 
+    train_parser = partial_part.add_parser(
+        "train", help="Do training without clustering", add_help=False
+    )
+    general_group = add_training_arguments(train_parser)
+    add_vae_arguments(train_parser)
+    train_parser.add_argument("--abundance_file", type=str, help="Input filename")
```

Comment:
there shouldn't be `add_argument` calls outside of functions that group them, like in `add_training_arguments` and `add_vae_arguments`. Please find where those are already defined or make a new grouping function